UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/RootBodyPartController.cs

```diff
@@ -14,49 +15,33 @@ public class RootBodyPartController : NetworkBehaviour
 	[SyncVar(hook = nameof(UpdateChildren))]
 	private string SerialisedNetIDs = "";
 
-	public List<RootBodyPartContainer> RootBodyParts = new List<RootBodyPartContainer>();
+	public List<uint> ToSerialised = new List<uint>();
```

Comment:
this stuff is changed in the Unity update

Reply:
understandable, however this list was only updated to discard the RootBodyParts